### ISSUE TITLE
the accuracy for retraining on the handgun dataset

### ISSUE THREAD

**Issue by nightzsky** [qfgaohao/pytorch-ssd#54]
Hi, may i know what is the accuracy that you have achieved for retraining on the handgun dataset? (the average precision for handgun class)

Thank you!

**Comment by qfgaohao** [comment]
@nightzsky not very high. I don't remember the exact number.

**Comment by nightzsky** [comment]
> @nightzsky not very high. I don't remember the exact number.

do you remember if it is >0.5?

**Comment by qfgaohao** [comment]
> > @nightzsky not very high. I don't remember the exact number.
> 
> do you remember if it is >0.5?

Not sure. A little suggestion, faster-rcnn might be better if accuracy is more than important speed.

**Comment by heimayouxia** [comment]
@qfgaohao May I asked how many epochs have you trained your pre_trained vgg16-ssd model? It is fairly accurate. But I can't achieve the comparable mAP on my custom dataset no matter how.

**Comment by qfgaohao** [comment]
@heimayouxia around 160-200 epoch on VOC? It also depends on the easiness of the dataset.
